go-controller/pkg/ovn/egressgw.go

```diff
@@ -83,15 +85,9 @@ func (oc *Controller) ensureRouteInfoLocked(podName ktypes.NamespacedName) (*ext
 
 	// 5. If routeInfo originally existed in the cache, there is a chance it was altered after we
```

Comment:
this comment is no longer accurate right? We arent checking if it was altered anymore, we are checking if it was deleted.

Reply:
good point, fixed.